.github/actions/pr-linting/index.js

```diff
@@ -0,0 +1,29 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+run();
+async function run() {
+  const title = core.getInput("pr-title")
+
+
+  const context = github.context;
+  if (context.payload.pull_request == null) {
+      core.setFailed('No pull request found.');
+      return;
+  }
+
+  core.info(
+    `🔎 Checking if the title of this PR "${title}" meets the requirements ...`
+  );
+
+  if ((/^\[?wip\]?/i).test(title)) {
```

Comment:
Do we want to validate the PR title's if they are WIP's? I figured for now we just skip but wanted to see other people's opinions